src/test/resources/application-test.yml

```diff
@@ -48,3 +48,5 @@ api:
   base:
     url:
       sdrs: http://localhost:8332
+      prison:
+        api: http://localhost:8333
```

Comment:
No auth here? I guess it's not needed yet (or your tests would fail 😆 )

Reply:
good spot @mikehalmamoj - I dont think I'll needto call  auth so will probably remove this in an upcoming PR.